templates/jv_change_register_now_button_text.php

```diff
@@ -0,0 +1,7 @@
+<?php
+// Remove the <?php tag if added to your functions.php
+// It changes the Register Now button string (Located on the Event page)
+add_filter( 'FHEE__EE_Ticket_Selector__display_ticket_selector_submit__btn_text', 'jv_change_register_now_text' );
+function jv_change_register_now_text() {
+	return 'Purchase now'; // Replace by your text
+}
```

Comment:
You can use this code instead of defining function.

```suggestion
add_filter( 'FHEE__EE_Ticket_Selector__display_ticket_selector_submit__btn_text', function () {
	return 'Purchase now'; // Replace by your text
} );

```
If you want to define a function then its name should be `jv_ee_change_register_now_text`. Note the to `_ee` prefix that I added to the function name.

Also please change the file name based on feedback of Tony.